ompi/communicator/ft/comm_ft.c

```diff
@@ -210,7 +222,9 @@ int ompi_comm_shrink_internal(ompi_communicator_t* comm, ompi_communicator_t** n
         exit_status = MPI_ERR_INTERN;
         goto cleanup;
     }
+#if OPAL_ENABLE_DEBUG
     stop = PMPI_Wtime();
+#endif
     OPAL_OUTPUT_VERBOSE((10, ompi_ftmpi_output_handle,
                          "%s ompi: comm_shrink: GRP COMPUTATION: %g seconds\n",
                          OMPI_NAME_PRINT(OMPI_PROC_MY_NAME), stop-start));
```

Comment:
I think we also need `#if OPAL_ENABLE_DEBUG` guards around the `OPAL_OUTPUT_VERBSE()` call which prints `stop-start`?